### ISSUE TITLE
"kyua help" doesn't provide any help

### ISSUE THREAD

**Issue by jmmv** [jmmv/kyua#36]
_From [LloydJParkes](https://code.google.com/u/116110872881215510874/) on July 13, 2012 18:06:11_

What steps will reproduce the problem? I followed the instructions on http://wiki.netbsd.org/kyua/ to setup Kyua after updating my local pkgsrc CVS tree. 

I then typed "kyua help" in an attempt to double check the version number of Kyua. What is the expected output? What do you see instead? I was hoping for some help, but I got this instead:
   kyua: E: Load of '/usr/pkg/etc/kyua/kyua.conf' failed: Failed to load Lua
   kyua: E: file '/usr/pkg/etc/kyua/kyua.conf': /usr/pkg/etc/kyua/kyua.conf:1:
   kyua: E: Invalid file format 'kyuafile'; expected 'config'. What version of the product are you using? On what operating system? I'm using Kyua 0.5 from pkgsrc on NetBSD 6.0_BETA2. Please provide any additional information below. I am complaining about the fact that a broken configuration will prevent the user from getting help, not the fact that the documentation doesn't match software, which is another issue.

_Original issue: http://code.google.com/p/kyua/issues/detail?id=36_

**Comment by jmmv** [comment]
_From [julio@meroh.net](https://code.google.com/u/116807652240575800029/) on July 13, 2012 15:59:38_

This is expected.  "help" needs to read the configuration file because the configuration will (at some point) define how the UI of the tool should look like.  (For example, whether lines should be, or not, automatically wrapped.  I haven't implemented this tunable yet though.)

But I agree that "help" should work regardless of configuration errors.  I think that changing "help" to report but ignore configuration errors would solve your problems?

**Status:** Accepted

**Comment by jmmv** [comment]
_From [LloydJParkes](https://code.google.com/u/116110872881215510874/) on July 13, 2012 16:19:19_

Possibly. I would have said yes, but mentioned that the UI is not defined yet, and so I don't know any more. I can say yes if I write a better bug report.

When I typed "kyua help", I was expecting to see some sort of usage information. I was expecting this because when I typed "kyua -v", it said " Type 'kyua help' for usage information." So, a solution that ensures that "kyua help" always displays a usage message in a similar style to the "Type 'kyua" message will work just fine.
